### PR TITLE
Issue #881 replace incorrect Windows command examples for cross-…

### PR DESCRIPTION
**Pull request by Preeticp**
…origin resource sharing with the correct ones.
Apart from this have made a few minor grammatical changes to avoid use of future tense and so on.

### CODE REVIEW

File: docs/source/using/interacting-with-openshift.adoc [hunk 1]
```diff
@@ -163,15 +163,18 @@ If you get the error `The specified patch need to be a valid JSON.` when you run
...
 ....
-$  .\minishift.exe --% openshift config set --patch "{\"routingConfig\": {\"subdomain\": \"192.168.99.101.nip.io\"}}"
+$ minishift openshift config set --patch '{\"corsAllowedOrigins\": [\".*\"]}'
 ....
 
 If you use Command Prompt you may need to use:
+
 ....
-minishift.exe openshift config set --patch "{\"routingConfig\": {\"subdomain\": \"192.168.99.101.nip.io\"}}"
+$ minishift openshift config set --patch "{\"corsAllowedOrigins\": [\".*\"]}"
```
Comment (agajdosi) [inline docs/source/using/interacting-with-openshift.adoc]: Command examples are in correct form.

File: docs/source/using/interacting-with-openshift.adoc [hunk 2]
```diff
@@ -46,7 +46,7 @@ for example, logging into another OpenShift instance, you can run:
 $ oc config use-context minishift
 ----
 
-For an introduction to `oc` usage, refer to the
+For an introduction to `oc` usage, see the
```
Comment (agajdosi) [inline docs/source/using/interacting-with-openshift.adoc]: I like the changes overall, text is more "friendly" now.

**Comment by budhrg** [comment]
LGTM 👍

File: docs/source/using/interacting-with-openshift.adoc (hunk 2 above)
Comment (Preeticp) [inline docs/source/using/interacting-with-openshift.adoc]: Thanks for the prompt review agajdosi.

File: docs/source/using/interacting-with-openshift.adoc (hunk 1 above)
Comment (LalatenduMohanty) [inline docs/source/using/interacting-with-openshift.adoc]: Nitpicking, the original doc had `minishift.exe` as it is for Windows. However `minishift` also works.

File: docs/source/using/interacting-with-openshift.adoc (hunk 2 above)
Comment (gbraad) [inline docs/source/using/interacting-with-openshift.adoc]: Well, is the location of the `oc` command still relevant now we have `minishift oc-env`?

File: docs/source/using/interacting-with-openshift.adoc (hunk 1 above)
Comment (gbraad) [inline docs/source/using/interacting-with-openshift.adoc]: This is better, as PowerShell does not specify the `.exe` extension

File: docs/source/using/interacting-with-openshift.adoc [hunk 3]
```diff
@@ -24,9 +24,9 @@ For details about managing the Minishift VM, see the link:../using/managing-mini
 The `minishift start` command creates an OpenShift instance using the
 https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md[cluster up] approach.
 
-For this purpose it copies the `oc` binary onto your host. You will find it under `~/.minishift/cache/oc/{openshift-version}/oc`, provided you use Minishift's default version of OpenShift.
+For this purpose it copies the `oc` binary onto your host. It is located in the  _~/.minishift/cache/oc/{openshift-version}_ directory, provided you use Minishift's default version of OpenShift.
```
Comment (gbraad) [inline docs/source/using/interacting-with-openshift.adoc]: Well, is the location of the oc command still relevant now we have minishift oc-env?

Comment (LalatenduMohanty) [inline docs/source/using/interacting-with-openshift.adoc]: @gbraad I think it is fine as it just point outs the location just for reference and then suggests to use `oc-env` command

**Pull request merged**